samples/Azure.ResourceManager.Sample/Generated/AvailabilitySetContainer.cs

```diff
@@ -5,23 +5,257 @@
+        {
+            using var scope = _clientDiagnostics.CreateScope("AvailabilitySetContainer.StartCreateOrUpdate");
+            scope.Start();
+            try
+            {
+                var originalResponse = Operations.CreateOrUpdate(Id.ResourceGroupName, availabilitySetName, parameters, cancellationToken: cancellationToken);
+                return new PhArmOperation<AvailabilitySet, AvailabilitySetData>(
+                originalResponse,
+                data => new AvailabilitySet(Parent, data));
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <inheritdoc />
+        /// <param name="availabilitySetName"> The name of the availability set. </param>
+        /// <param name="parameters"> Parameters supplied to the Create Availability Set operation. </param>
+        /// <param name="cancellationToken"> A token to allow the caller to cancel the call to the service. The default value is <see cref="P:System.Threading.CancellationToken.None" />. </param>
+        public async Task<ArmOperation<AvailabilitySet>> StartCreateOrUpdateAsync(string availabilitySetName, AvailabilitySetData parameters, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("AvailabilitySetContainer.StartCreateOrUpdateAsync");
+            scope.Start();
+            try
+            {
+                var originalResponse = await Operations.CreateOrUpdateAsync(Id.ResourceGroupName, availabilitySetName, parameters, cancellationToken: cancellationToken).ConfigureAwait(false);
+                return new PhArmOperation<AvailabilitySet, AvailabilitySetData>(
+                originalResponse,
+                data => new AvailabilitySet(Parent, data));
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <inheritdoc />
+        /// <param name="availabilitySetName"> The name of the availability set. </param>
+        /// <param name="cancellationToken"> A token to allow the caller to cancel the call to the service. The default value is <see cref="P:System.Threading.CancellationToken.None" />. </param>
+        public override ArmResponse<AvailabilitySet> Get(string availabilitySetName, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("AvailabilitySetContainer.Get");
+            scope.Start();
+            try
+            {
+                return new PhArmResponse<AvailabilitySet, AvailabilitySetData>(
+                Operations.Get(Id.ResourceGroupName, availabilitySetName, cancellationToken: cancellationToken),
+                data => new AvailabilitySet(Parent, data));
```

Comment:
```suggestion
                var response = Operations.Get(Id.ResourceGroupName, availabilitySetName, cancellationToken: cancellationToken);
                ArmResponse.FromValue(new AvailabilitySet(Parent, response.Value), response.GetRawResponse());
```
When `ArmResponse.FromValue` is ready